CMakeLists.txt

```diff
@@ -182,6 +194,13 @@ configure_file(
   ${KWIVER_SETUP_SCRIPT_FILE}
   @ONLY
   )
+if(WIN32)
+  configure_file(
+    ${KWIVER_CMAKE_DIR}/setup_KWIVER.bat.in
+    ${KWIVER_SETUP_BATCH_FILE}
+    @ONLY
+    )
+endif()
 
 # install set up script
 option( KWIVER_INSTALL_SET_UP_SCRIPT "Whether or not to install setup_KWIVER.sh" ON )
```

Comment:
Since this option applies to both the batch file and shell script, maybe the '.sh' could be dropped.